exercises/01_trim_right/01_trim_right_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import "testing"
+
+func TestTrimRight(t *testing.T) {
+	for _, tc := range []struct {
+		name  string
+		input string
+		want  string
+	}{
+		{"empty", "", ""},
```

Comment:
I also recommend adding tests like this:
- `"a"` just a single character
- `" "` just a seingle whitespace
- `"a "` a single whitespace and a character
There are often bugs that could be found by these simple corner cases. Your implementation shouldn't fail for those though. It seems good.